package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "ep_new_pad",
-  "version": "0.0.1",
-  "description": "Etherpad lite plugin to crate a REALLY new pad (database check). Call http://example.com/randomPad to use it.",
+  "version": "0.0.2,
```

Comment:
Here is one " missing. The rest looks good to me :-)

Reply:
oops! fixed in 9aa09f2f002f9f3885b317220910fa4041a958be